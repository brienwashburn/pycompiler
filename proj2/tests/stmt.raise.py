raise
raise a
raise bar from foo
