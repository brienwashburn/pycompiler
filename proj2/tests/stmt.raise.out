(Module (Raise) (Raise (Name a)) (Raise (Name bar) (Name foo)))
